core/src/main/java/dk/alexandra/fresco/lib/statistics/DeaSolver.java

```diff
@@ -22,9 +22,6 @@
 import java.util.stream.IntStream;
 
 /**
- * NativeProtocol for solving DEA problems.
- *
- * <p>
  * Given a dataset (two matrices of inputs and outputs) and a number of query vectors, the protocol
```

Comment:
Shouldn't there be a `<p>` tag here?

Reply:
I removed the header stating the wrong intention.